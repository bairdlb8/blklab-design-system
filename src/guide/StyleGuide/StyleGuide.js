import React from 'react'
import PropTypes from 'prop-types'

import './StyleGuide.scss'

const StyleGuide = ({className=''}) => {
  return <div>StyleGuide</div>
}

StyleGuide.propTypes = {}

export default StyleGuide
