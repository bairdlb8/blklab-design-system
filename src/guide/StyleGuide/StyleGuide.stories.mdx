import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import StyleGuide from './StyleGuide'

<Meta title="Guide / StyleGuide" component={ StyleGuide } />

# StyleGuide

## Add some text here

export const Template = (args) => {
  return <StyleGuide {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ StyleGuide } />
