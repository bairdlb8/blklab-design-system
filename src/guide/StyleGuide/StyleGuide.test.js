import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'
import StyleGuide from './StyleGuide'

expect.extend(toHaveNoViolations)

describe('<StyleGuide />', () => {
  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<StyleGuide />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<StyleGuide />)

    expect(asFragment(<StyleGuide />)).toMatchSnapshot()
  })
})
