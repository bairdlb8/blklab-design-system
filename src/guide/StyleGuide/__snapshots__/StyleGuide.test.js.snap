// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyleGuide /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    StyleGuide
  </div>
</DocumentFragment>
`;
